### DISCUSSION TITLE
Release 5.0.0 Sunbird ED Regression Status

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Ed/Community#114] [General Discussion]
Hi All,

Please find the testing status of R5.0.0 as of 30th Aug 2022

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress  (27.20% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)

**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Planned Vs Actual Testing Status:**

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  

</body>
</html>

Execution Status:

![image](https://user-images.githubusercontent.com/40622169/187451630-435f0126-9e44-4a1c-abcb-e6939ff43690.png)

Defects Status:

![image](https://user-images.githubusercontent.com/40622169/187451800-2ac85652-fecd-41f3-a78c-4c842388074e.png)

![image](https://user-images.githubusercontent.com/40622169/187451898-f269cc24-e0bd-4d96-b7a5-756bc7dbb1f1.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 01st Sep  2022

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (37.20% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Planned Vs Actual Testing Status:

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  


</body>
</html>

Execution Status:

![image](https://user-images.githubusercontent.com/40622169/187935805-3f3ea6e5-1a07-47cd-bfc8-01deba7b2ff8.png)

Defects Status:

![image](https://user-images.githubusercontent.com/40622169/187935933-d3cadd44-8d9f-4f87-a626-5dd5baaa4a60.png)

![image](https://user-images.githubusercontent.com/40622169/187936017-3ccac1e1-41b6-4e49-baf5-8d2bbed23ea3.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @manju4201

**Reply by ShailandraOjha** [reply]
Hi All,

Please find the testing status of R5.0.0 as of 02nd Sep 2022

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (37.20% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Planned Vs Actual Testing Status:**
![image](https://user-images.githubusercontent.com/44188853/188193679-9473e632-6573-41a5-8622-ed7b1017f3ef.png)

**Execution Status:**
![image](https://user-images.githubusercontent.com/44188853/188193852-75a12527-73f4-4cae-ab99-7998308fed38.png)

**Defects Status:**
![image](https://user-images.githubusercontent.com/44188853/188194024-479de2b5-a4e1-412c-ac79-833604b3fb2a.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @anunagaraj @@swayangjit

Regards,
Shailandra Ojha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 05th Sep 2022

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (51.90% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Planned Vs Actual Testing Status:**
<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  
02-09-2022 | 42.00% | 45.00% | -Regression Execution is in Progress-There are 38 Open Defects |  
05-09-2022 | 50.00% | 51.90% | -Regression Execution is in Progress- 2 QA team members were  on Sick leave-There are 32 Open Defects |  


</body>
</html>

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/188462448-ffb63b32-76a6-40b6-a589-ca97add358d0.png)

**Defects Status:**
![image](https://user-images.githubusercontent.com/40622169/188462631-ca6481d4-99c7-46f3-917f-074a161ac79d.png)

![image](https://user-images.githubusercontent.com/40622169/188462703-ea0f516f-816d-41f6-8124-f2959dd1d22b.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @ShailandraOjha  @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 06th Sep 2022

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress (60% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Planned Vs Actual Testing Status:**
![image](https://user-images.githubusercontent.com/40622169/188658678-c0431f09-0556-4172-b614-08b84ca16d10.png)

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/188658863-8d7e3f97-b4ac-446b-aedb-786ea9105d50.png)

**Defects Status:**

![image](https://user-images.githubusercontent.com/40622169/188659132-2eea1c3d-ede4-49e4-a2e5-1052b1679b8b.png)

![image](https://user-images.githubusercontent.com/40622169/188659214-bd2ae0d3-aae8-4794-b6ac-53e98a4de3dd.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 07th Sep 2022

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress (67.8% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Planned Vs Actual Testing Status:**
<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  
02-09-2022 | 42.00% | 45.00% | -Regression Execution is in Progress-There are 38 Open Defects |  
05-09-2022 | 50.00% | 51.90% | -Regression Execution is in Progress- 2 QA team members were  on Sick leave-There are 32 Open Defects |  
06-09-2022 | 58.33% | 60.00% | -Regression Execution is in Progress-There are 32 Open Defects |  
07-09-2022 | 67.00% | 67.80% | -Regression Execution is in Progress-There are 32 Open Defects | SB-30816 - Blocked with 103 test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases


</body>
</html>

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/188940190-e5b9c2af-9a57-47ea-bdfa-ef9251b10544.png)

**Defect Status**
![image](https://user-images.githubusercontent.com/40622169/188940280-d9047554-872c-4b44-b8f5-ca220814d83c.png)

![image](https://user-images.githubusercontent.com/40622169/188940343-718f5502-9a55-4bd4-843e-43ce5eca8105.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @ShailandraOjha @swayangjit  @tarang27


Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 08th Sep 2022

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress (73.8% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Planned Vs Actual Testing Status:**
<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  
02-09-2022 | 42.00% | 45.00% | -Regression Execution is in Progress-There are 38 Open Defects |  
05-09-2022 | 50.00% | 51.90% | -Regression Execution is in Progress- 2 QA team members were  on Sick leave-There are 32 Open Defects |  
06-09-2022 | 58.33% | 60.00% | -Regression Execution is in Progress-There are 32 Open Defects |  
07-09-2022 | 67.00% | 67.80% | -Regression Execution is in Progress-There are 32 Open Defects | SB-30816 - Blocked with 103 test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
08-09-2022 | 75.00% | 73.80% | -Regression Execution is in Progress- 1 QA member is on Sick Leave - There are 39 Open Defects | **SB-30816** - Blocked with 103 test cases  **SB-30924 -** Blocked with 25 Test cases **SB-30814** - Blocked with 38 test cases **SB-30881 -** Blocked with 21 test cases

</body>
</html>

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/189176901-bbd185a4-492e-4046-a66a-76b4f2aa8c42.png)

**Defect Status:**
![image](https://user-images.githubusercontent.com/40622169/189177031-72ca6075-68c8-4704-ba2e-b2e9d91e13ea.png)

![image](https://user-images.githubusercontent.com/40622169/189177091-ccc9a339-e274-4b93-846a-478d7ac2ae65.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @ShailandraOjha @swayangjit @tarang27

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 09th Sep 2022

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (82.3% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Planned Vs Actual Testing Status:
<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  
02-09-2022 | 42.00% | 45.00% | -Regression Execution is in Progress-There are 38 Open Defects |  
05-09-2022 | 50.00% | 51.90% | -Regression Execution is in Progress- 2 QA team members were  on Sick leave-There are 32 Open Defects |  
06-09-2022 | 58.33% | 60.00% | -Regression Execution is in Progress-There are 32 Open Defects |  
07-09-2022 | 67.00% | 67.80% | -Regression Execution is in Progress-There are 32 Open Defects | SB-30816 - Blocked with 103 test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
08-09-2022 | 75.00% | 73.80% | -Regression Execution is in Progress- 1 QA member is on Sick Leave- There are 39 Open Defects | SB-30816 - Blocked with 103 test cases SB-30924 - Blocked with 25 Test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
09-09-2022 | 83.00% | 82.30% | -Regression Execution is in Progress- There are 33 Open Defects | SB-30924 - Blocked with 25 Test cases SB-30814 - Blocked with 38 test cases


</body>
</html>

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/189385666-648d1aca-9294-48e3-817f-2739a81551a1.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/189385952-11d8c82e-f8a2-4cbb-94e3-ea66484a7860.png)

![image](https://user-images.githubusercontent.com/40622169/189386035-a2c5a38a-4271-4d6e-ac5d-e4deab800619.png)


@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @ShailandraOjha @swayangjit @tarang27

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 12th Sep 2022

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (91.30% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Planned Vs Actual Testing Status:
<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  
02-09-2022 | 42.00% | 45.00% | -Regression Execution is in Progress-There are 38 Open Defects |  
05-09-2022 | 50.00% | 51.90% | -Regression Execution is in Progress- 2 QA team members were  on Sick leave-There are 32 Open Defects |  
06-09-2022 | 58.33% | 60.00% | -Regression Execution is in Progress-There are 32 Open Defects |  
07-09-2022 | 67.00% | 67.80% | -Regression Execution is in Progress-There are 32 Open Defects | SB-30816 - Blocked with 103 test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
08-09-2022 | 75.00% | 73.80% | -Regression Execution is in Progress- 1 QA member is on Sick Leave- There are 39 Open Defects | SB-30816 - Blocked with 103 test cases SB-30924 - Blocked with 25 Test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
09-09-2022 | 83.00% | 82.30% | -Regression Execution is in Progress- There are 33 Open Defects | SB-30924 - Blocked with 25 Test casesSB-30814 - Blocked with 38 test cases
12-09-2022 | 92.00% | 91.30% | -Regression Execution is in Progress- There are 28 Open Defects |  


</body>
</html>

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/189702501-95efafdd-f1ba-4bfe-b6da-1416040ddcde.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/189702750-15e7ef64-050a-4da9-a1c8-d71baffd6513.png)

![image](https://user-images.githubusercontent.com/40622169/189702874-18e31e80-372e-4353-8033-031b1a4f119d.png)


@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @ShailandraOjha @swayangjit @tarang27

Regards,
Anusha

**Comment by ShailandraOjha** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 13th Sep 2022

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (96% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Note: We are left with 130 test cases and 12 open issues hence we will not be able to provide the sign off today.**

**Planned Vs Actual Testing Status:**
<html>
<body>

Date | Planned Completion % | Actual Completion % | Remarks | Blocker Defects
-- | -- | -- | -- | --
26-08-2022 | 8.33% | 9.30% | -Regression Execution is in Progress |  
29-08-2022 | 16.00% | 17.60% | -Regression Execution is in Progress-There are 16 Open Defects | SB-30816 - Blocked with 10 Test cases
30-08-2022 | 25.00% | 27.20% | -Regression Execution is in Progress-There are 22 Open Defects |  
01-09-2022 | 33.00% | 37.20% | -Regression Execution is in Progress-There are 24 Open Defects |  
02-09-2022 | 42.00% | 45.00% | -Regression Execution is in Progress-There are 38 Open Defects |  
05-09-2022 | 50.00% | 51.90% | -Regression Execution is in Progress- 2 QA team members were  on Sick leave-There are 32 Open Defects |  
06-09-2022 | 58.33% | 60.00% | -Regression Execution is in Progress-There are 32 Open Defects |  
07-09-2022 | 67.00% | 67.80% | -Regression Execution is in Progress-There are 32 Open Defects | SB-30816 - Blocked with 103 test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
08-09-2022 | 75.00% | 73.80% | -Regression Execution is in Progress- 1 QA member is on Sick Leave- There are 39 Open Defects | SB-30816 - Blocked with 103 test cases SB-30924 - Blocked with 25 Test casesSB-30814 - Blocked with 38 test casesSB-30881 - Blocked with 21 test cases
09-09-2022 | 83.00% | 82.30% | -Regression Execution is in Progress- There are 33 Open Defects | SB-30924 - Blocked with 25 Test casesSB-30814 - Blocked with 38 test cases
12-09-2022 | 92.00% | 91.30% | -Regression Execution is in Progress- There are 28 Open Defects |  
13-09-2022 | 100.00% | 97.00% | - 130 test cases are failed and blocked due to open issues.- There are 12 open issues. |  


</body>
</html>

**Execution Status:**
![image](https://user-images.githubusercontent.com/44188853/189938386-590ca0fa-8b4d-41a8-8f3f-431570e0e948.png)

**Defect Status:**
![image](https://user-images.githubusercontent.com/44188853/189938525-123f252d-9fdf-487c-8f1b-e13982f34914.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai @madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @swayangjit @tarang27

Reagrds,
Shailandra Ojha
